steering_vectors/aggregators.py

```diff
@@ -30,6 +53,23 @@ def pca_aggregator(pos_acts: Tensor, neg_acts: Tensor) -> Tensor:
     return sign * vec
 
 
+def _get_normalized_regression_coef(
+    pos_acts: Tensor, neg_acts: Tensor, regression: Regression
+) -> Tensor:
+    reg = regression.fit(
+        torch.cat([pos_acts, neg_acts]).cpu().to(torch.float32).numpy(),
+        torch.cat([torch.ones(pos_acts.shape[0]), -1 * torch.ones(neg_acts.shape[0])])
+        .cpu()
+        .to(torch.float32)
+        .numpy(),
+    )
+
+    coef = torch.tensor([reg.coef_])
+    normalized_coef = F.normalize(coef, dim=0)
+
+    return normalized_coef
```

Comment:
we should make sure to  move this tensor to the same device as the input tensors, and match the dtype as well

Reply:
Implemented by [7ba358f](https://github.com/steering-vectors/steering-vectors/pull/22/commits/7ba358f8229cbe662932a7d35cffe92cefbf6e28)